General improvements to ONT pages and add new ONTs (#63)
hack-gpon/hack-gpon.github.io: _ont/ont-zte-f6005.md

| Vendor      | ZTE                               |
| Model       | F6005                             |
| Chipset     | Realtek RTL9601D                  |
| Manufacter  | [CIG G-97CP](/ont-cig-g-97cp)     |
| Flash       |                                   |
| RAM         |                                   |
| System      | Linux (Luna SDK 1.9.0)            |
| HSGMII      | Yes                               |
| Optics      | SC/APC                            |
| IP address  | 192.168.1.1                       |
| Web Gui     | ✅ user `admin`, password `admin` |

> | Telnet      | ✅ user `ont` password should be calculated, calculated by hash of a username and serial number |

Reviewer: ```suggestion
| Telnet      | ✅ user `ont`, password should be computed based on a hash of the username and the serial number |
```

## List of partitions
## List of firmwares and files

# Telnet

> To calculate the password for telnet access, you can use this tool, simply by entering the S/N

Reviewer: ```suggestion
You can use this tool to generate the password for telnet access
```
